### PR TITLE
postgres blog post

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: apps/www/_blog/2022-10-03-postgres-wasm.mdx [hunk 1]
```diff
@@ -0,0 +1,253 @@
+---
+title: Postgres WASM by Snaplet and Supabase
```
Comment (RamiroND) [inline apps/www/_blog/2022-10-03-postgres-wasm.mdx:2]: @kiwicopple  Should we add Open Source to the title? So: Open source Postgres WASM by Snaplet and Supabase

File: apps/www/_blog/2022-10-03-postgres-wasm.mdx [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+description: Turns out the question of which identifier to use as a Primary Key is complicated -- we're going to dive into some of the complexity and inherent trade-offs, and figure things out
```
Comment (J0) [inline apps/www/_blog/2022-10-03-postgres-wasm.mdx:3]: Sanity check: is this the intended description? Seems like the description for the prev blog post

File: apps/www/_blog/2022-10-03-postgres-wasm.mdx [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+- **Offline data:** running it in the browser for an offline cache, similar to [sql.js](https://sql.js.org/#/) or [absurd-sql](https://github.com/jlongster/absurd-sql).
+- **Offline data analysis:** using it in a dashboard for offline data analysis and charts.
+- **Testing:** testing PostgresSQL functions, triggers, data modeling, logical replication, etc.
+- **Dev environments:** use it as a development environment — pull data from production or push new data, functions, triggers, views up to production.
+- **Snapshots:** create a test version of your database with sample data, then take a snapshot to send to other developers.
+- **Support:** send snapshots of your database to support personnel to demonstrate an issue you're having.
+
+## Repository Overview
+
+We've divided in our repository into three folders: a virtual machine, a web applications, and a network proxy.
```
Comment (J0) [inline apps/www/_blog/2022-10-03-postgres-wasm.mdx:89]:
```suggestion
We've divided in our repository into three folders: a virtual machine, a web application, and a network proxy.
```

/nit

File: apps/www/_blog/2022-10-03-postgres-wasm.mdx (hunk 2 above)
Comment (RamiroND) [reply apps/www/_blog/2022-10-03-postgres-wasm.mdx:3]: It's not yeah. Will update it.

File: apps/www/_blog/2022-10-03-postgres-wasm.mdx [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+Once the emulator is running, in the words of Fabian, “it runs as a black box process”, and the only way to communicate between the
+UI and the operating system running in the emulator is through `serial0_send` to send keystrokes over the emulated serial port,
+or through the `create_file` function which uploads a file to the running emulator.
+
+To restart the network after restoring a state file, we need to run a short script to unbind and re-bind the emulated network card
+(so it gets a new mac address and thus a unique private IP address at the proxy), and then a standard network restart command.
+We could run this script by sending keystrokes to the emulator, but that's clumsy. What if the user is at a `psql` prompt? Then we'd send `\! script_command`.
+But if they're at an OS prompt, that won't work.
+
+We solved by creating a folder called `\inbox` and added a listener to the folder. When files arrive there, if the file ends in `.sh` (a shell script),
```
Comment (J0) [inline apps/www/_blog/2022-10-03-postgres-wasm.mdx:216]:
```suggestion
We solved the issue by creating a folder called `\inbox` and added a listener to the folder. When files arrive there, if the file ends in `.sh` (a shell script),
```

File: apps/www/_blog/2022-10-03-postgres-wasm.mdx (hunk 1 above)
Comment (kiwicopple) [reply apps/www/_blog/2022-10-03-postgres-wasm.mdx:2]: I think better as it is - this title is enough to pique the interest and they will see it's open source from the first paragraph

**Review by RamiroND — APPROVED** [review]

**Pull request merged**